zsl/testing/http.py

```diff
@@ -41,8 +46,8 @@ def requestTask(self, client, task, data, headers=None):
         :param task: Url which will be requested using POST method.
         :param data: Data which will be posted and first converted to JSON.
         :param headers: Dictionary of headers that'll be appended the 
```

Comment:
Should be there an '\' at the end of the line for param help string, when you wrap the line, so the sphinx includes it with it?